src/current/_includes/releases/v24.1/v24.1.0.md

```diff
@@ -0,0 +1,274 @@
+        </tr>
+        <tr>
+            <th colspan="1" rowspan="1">Ver.</th>
+            <th colspan="1" rowspan="1" style="white-space: nowrap;">Self-Hosted</th>
+            <th colspan="1" rowspan="1">Dedicated</th>
+            <th colspan="1" rowspan="1">Serverless</th>
+        </tr>
+    </thead>
+    <tbody>
+        <!-- Rows for each feature -->
+        <tr>
+        <td>
+            <p class="feature-summary">TBD</p>
+            <p class="feature-description">TBD</p>
+        </td>
+        <td>TBD</td>
+        <td class="icon-center">{% include icon-no.html %}</td>
+        <td class="icon-center">{% include icon-yes.html %}</td>
+        <td class="icon-center">{% include icon-no.html %}</td>
+        </tr>
+    </tbody>
+</table>
+
+<table>
+    <thead>
+        <tr>
+            <th id="feature-detail-key" colspan="2">Feature detail key</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>*</td>
+            <td>Features marked “All*” were recently made available in the CockroachDB Cloud platform. They are available for all supported versions of CockroachDB, under the deployment methods specified in their row under Availability.</td>
+        </tr>
+        <tr>
+            <td>**</td>
+            <td>Features marked “All**” were recently made available via migration tools maintained outside of the CockroachDB binary. They are available to use with all supported versions of CockroachDB, under the deployment methods specified in their row under Availability.</td>
+        </tr>
+        <tr>
+            <td>{% include icon-yes.html %}</td>
+            <td>Feature is available for this deployment method of CockroachDB as specified in the icon’s column: CockroachDB Self-Hosted, CockroachDB Dedicated, or CockroachDB Serverless.</td>
+        </tr>
+        <tr>
+            <td>{% include icon-no.html %}</td>
+            <td>Feature is not available for this deployment method of CockroachDB as specified in the icon’s column: CockroachDB Self-Hosted, CockroachDB Dedicated, or CockroachDB Serverless.</td>
+        </tr>
+    </tbody>
+</table>
+
+
+</div>
+
+<h4 id="v24-1-0-backward-incompatible-changes">Backward-incompatible changes</h4>
+
+Before [upgrading to CockroachDB v24.1]({% link v24.1/upgrade-cockroach-version.md %}), be sure to review the following backward-incompatible changes, as well as [key cluster setting changes](#v24-1-0-cluster-settings), and adjust your deployment as necessary.
+
+- TBD
+
+<h4 id="v24-1-0-cluster-settings">Key Cluster Setting Changes</h4>
+
+The following changes should be reviewed prior to upgrading. Default cluster settings will be used unless you have manually set a value for a setting. This can be confirmed by checking the `system.settings` table (`select * from system.settings`) to view the non-default settings.
+
+- TBD
+
+<h4 id="v24-1-0-deprecations">Deprecations</h4>
+
+{% comment %}TODO: Intro para? Each sibling section has one.{% endcomment %}
```

Comment:
I'm good to leave this comment for now, so we come back to it, though not critical for the 24.1 release.